### DISCUSSION TITLE
New Piccolo tutorial video

### DISCUSSION THREAD

**Discussion by dantownsend** [piccolo-orm/piccolo#199] [General]
I just published a new video, which covers creating a Piccolo project, and how Piccolo apps work:

https://www.youtube.com/watch?v=vciBNQapoFk

There's a bunch more videos I'd like to make. If there's anything in particular people think would be useful, let me know.

**Comment by sinisaos** [comment]
@dantownsend This is great and much more detailed than the old videos. When I first used Piccolo, the video tutorial helped me a lot and I think the video guide is a great way for people to get to know the library and start using it.

**Reply by dantownsend** [reply]
Thanks! I agree - watching videos is a great way to learn.

I just uploaded another video covering `piccolo sql_shell run` and `piccolo shell run`. https://www.youtube.com/watch?v=NvLFN6Gg7b8

I've got a list of about 20 Piccolo videos I'd like to make when I have time!

**Reply by sinisaos** [reply]
That's very cool and useful.

**Comment by aminalaee** [comment]
@dantownsend That's great, I think if we extend the documentation, that would be great for new users too. We could have more explanation and examples and maybe add some missing points.

**Reply by dantownsend** [reply]
I agree - the docs are lacking in some areas.
